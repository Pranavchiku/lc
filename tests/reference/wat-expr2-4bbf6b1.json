{
    "basename": "wat-expr2-4bbf6b1",
    "cmd": "lc --no-color --show-wat {infile} -extra-arg=\"-Isrc/runtime/include\"",
    "infile": "tests/expr2.c",
    "infile_hash": "11cb114715d67a4763454167abc900089b2119ecf18a980d18a30322",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "wat-expr2-4bbf6b1.stdout",
    "stdout_hash": "b7a97558339a290cf15d8042b51d4f11ee8460bca568996113d213cb",
    "stderr": "wat-expr2-4bbf6b1.stderr",
    "stderr_hash": "66dd58d266fcb5d97d685b1bc0b99d94eb015b993ba5cef4853fbe1b",
    "returncode": 0
}