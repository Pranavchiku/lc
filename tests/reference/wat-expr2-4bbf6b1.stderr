Error while trying to load a compilation database:
Could not auto-detect compilation database for file "tests/expr2.c"
No compilation database found in $DIR/tests or any parent directory
fixed-compilation-database: Error while opening fixed database: No such file or directory
json-compilation-database: Error while opening JSON database: No such file or directory
Running without flags.
[0;33;1mwarning[0;0m[0;1m: Pointers are not currently supported[0;0m
 [0;34;1m-->[0;0m tests/expr2.c:5:1
  [0;34;1m|[0;0m
[0;34;1m5 |[0;0m     x = (2+3)*5;
  [0;34;1m|[0;0m [0;33;1m^^^^^^^^^^^^ emitting integer for now[0;0m


[0;1mNote[0;0m: Please report unclear or confusing messages as bugs at
https://github.com/lfortran/lfortran/issues.
